src/lib.rs

```diff
@@ -62,6 +63,17 @@ impl Config {
             assets,
         })
     }
+
+    /// Returns the directory where output files should be placed
+    pub fn output_path(&self) -> PathBuf {
+        self.manifest_path.join("target").join("doc")
+    }
+
+    /// Open the generated docs in a web browser.
+    pub fn open_docs(&self) -> Result<()> {
+        open::that(self.output_path().join("index.html")).unwrap(); // TODO: Error handling
```

Comment:
@mbrubeck doesn't look like you actually put the change in for this at all since error.rs never got updated with the error type and the unwrap is still here.